client-src/elements/chromedash-toast.ts

```diff
@@ -1,27 +1,24 @@
 import {LitElement, css, html} from 'lit';
 import {SHARED_STYLES} from '../css/shared-css.js';
+import {property} from 'lit/decorators.js';
 
 const DEFAULT_DURATION = 7000;
 
 class ChromedashToast extends LitElement {
-  static get properties() {
-    return {
-      msg: {type: String},
-      open: {type: Boolean, reflect: true},
-      actionLabel: {attribute: false},
-      currentTimeout: {type: Number},
-      waitingForTransition: {type: Boolean},
-    };
-  }
+  @property({type: String})
+  msg = '';
 
-  constructor() {
-    super();
-    this.msg = '';
-    this.actionLabel = '';
-    this.open = false;
-    this.currentTimeout = null;
-    this.waitingForTransition = false;
-  }
+  @property({type: Boolean, reflect: true})
+  open = false;
+
+  @property({attribute: false})
+  actionLabel = '';
+
+  @property({type: Number})
+  currentTimeout: number | null = null;
```

Comment:
Looking at the code using this, it seems like it shouldn't be settable using an attribute on the element, and instead should be 
```suggestion
  @state
  currentTimeout: number | null = null;
```
But I haven't checked all the calling code to make sure that's a correct change, and you've correctly translated the `static properties()` method.